.gitmodules

```diff
@@ -82,3 +82,13 @@
 	path = sources/percona-images
 	url = https://github.com/Percona-Lab/percona-images
 	branch = master
+
+## PMM Tests
+[submodule "pmm-api-tests"]
+	path = sources/pmm-api-tests/src/github.com/Percona-Lab/pmm-api-tests
+	url = https://github.com/Percona-Lab/pmm-api-tests
+	branch = master
+[submodule "pmm-qa"]
+	path = sources/pmm-qa/src/github.com/percona/pmm-qa
```

Comment:
We need those long paths for Go code (for GOPATH). Do we need it there? Can we simplify it to `sources/pmm-qa`?

Reply:
I did that exactly during the initial commit and you requested to change to GOPATH :confused: so for this iteration, we can keep it same, I will merge change in next PR with test suite

Reply:
We need GOPATH for github.com/Percona-Lab/pmm-api-tests above. I think I requested it for that repo.